tests/viz/test_exporter.py

```diff
@@ -137,9 +149,9 @@ def test_single_subdomains(setup: ExporterTestSetup, subdomain: SingleSubdomain)
     sd = subdomain.grid
     # Define data
     dummy_scalar = np.ones(sd.num_cells) * sd.dim
-    dummy_scalar_pt = np.ones(sd.num_nodes) * sd.dim
+    dummy_scalar_pt = np.ones(num_nodes(sd)) * sd.dim
```

Comment:
From that test, I understand, we have two possibilities to export 0d grid data. Using point data and cell data?

Reply:
Yes